ledger/error.go

```diff
@@ -33,6 +33,19 @@ func (tile TransactionInLedgerError) Error() string {
 	return fmt.Sprintf("transaction already in ledger: %v", tile.Txid)
 }
 
+// LeaseInLedgerError is returned when a transaction cannot be added because it has a lease that already being used in the relavant rounds
+type LeaseInLedgerError struct {
+	txid  transactions.Txid
+	lease txlease
+}
+
+// Error implements the error interface for the LeaseInLedgerError stuct
+func (lile LeaseInLedgerError) Error() string {
```

Comment:
`lile LeaseInLedgerError` to reduce copying?

Reply:
not sure what you mean by saying that. Are you asking if I can convert that to a `func (lile *LeaseInLedgerError) Error() string` so that the data members would not need to be copied ?

Reply:
sorry, yes